app/routes/events/components/EventAdministrate/Attendees.js

```diff
@@ -144,7 +144,7 @@ export default class Attendees extends Component<Props, State> {
             </Link>
           </h2>
           {event.useContactTracing &&
-            currentUser.id == event.createdBy &&
+            currentUser.id === event.createdBy &&
```

Comment:
Btw. I think `createdBy` can be an object with an user some time. Maybe outside the scope of this PR tho.

Reply:
Hmm, I could just do ```(currentUser.id === event.createdBy || currentUser.id ===.event.createdBy.id)``` to cover both cases. 
I think it is a bit out of scope to make event.createdBy to always be either a `User` or a number. :thinking: